images/daemon/pkg/daemon/kata_openshift.go

```diff
@@ -406,9 +406,18 @@ func installRPMs(k *KataOpenShift) (err error) {
 		fmt.Println(err)
 	}
 
-	srcRef, err := alltransports.ParseImageName("docker://quay.io/isolatedcontainers/kata-operator-payload:" + k.PayloadTag)
+	payloadImage := os.Getenv("KATA_PAYLOAD_IMAGE")
+	if payloadImage == "" {
+		payloadImage = "docker://quay.io/isolatedcontainers/kata-operator-payload:" + k.PayloadTag
+	} else {
+		log.Println("WARNING: kataconfig installation is tainted")
```

Comment:
What is meant by tainted here?

Reply:
Same meaning it has when you load a proprietary kernel module. You're on your own when problems occur. 
https://www.kernel.org/doc/html/latest//admin-guide/tainted-kernels.html